e2e-tests/tests/test_image_redactor.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from common.methods import image_redactor
+
+
+def test_given_image_with_int_color_filling_then_we_get_redact_image():
+    # gray redact
+    expected_result_image = get_file("gray_redact.png")
+    response = image_redactor("original_image.png", 255)
+    validate_and_close(expected_result_image, response)
+
+
+def test_given_image_with_tuple_color_filling_then_we_get_redact_image():
+    # purple redact
+    expected_result_image = get_file("purple_redact.png")
+    response = image_redactor("original_image.png", "102, 0, 102")
+    validate_and_close(expected_result_image, response)
+
+
+def test_given_image_with_no_color_filling_then_we_get_redact_image():
```

Comment:
I think that one happy scenario is enough if we have those tests inside the service

Reply:
You are missing @api marks on all tests

Reply:
Left only one happy scenario and added the api mark